tests/testthat/test-addin.R

```diff
@@ -0,0 +1,43 @@
+context("use_addin")
+
+test_that("use_addin() creates the first addins.dcf as promised, default", {
+  scoped_temporary_package()
+  use_addin()
+
+  addin_dcf <- readLines(proj_path("inst", "rstudio", "addins.dcf"))
+  expect_true(
+    all(
+      c("Name: New Addin Name", "Description: New Addin Description",
+        "Binding: new_addin", "Interactive: false") %in% addin_dcf
+    )
+  )
+})
+
+test_that("use_addin() creates the first addins.dcf as promised, option", {
+  scoped_temporary_package()
+  use_addin("addin.test")
+
+  addin_dcf <- readLines(proj_path("inst", "rstudio", "addins.dcf"))
+  expect_true(
+    all(
+      c("Name: New Addin Name", "Description: New Addin Description",
+        "Binding: addin.test", "Interactive: false") %in% addin_dcf
+    )
+  )
+})
+
+test_that("use_addin() can create bindings for multiple addins.", {
```

Comment:
Is this necessary? Do you perceive some potential problem with creating multiple addins?

Reply:
I added this piece because I added an `append` option to `write_utf8` so it can do "appending". I guess this test here is not "unit" enough. Instead, I should have just written a simple test with `write_utf8`.